packages/manager/apps/procedures/src/components/FileInput/FileInputList.tsx

```diff
@@ -26,9 +27,11 @@ export const FileInputList: FunctionComponent<Props> = ({ className }) => {
   return value?.length ? (
     <div className={className}>
       {multiple && (
-        <label className="text-sky-900 text-xs font-semibold">
-          {t('account-disable-2fa-file-input-attachments')}
-        </label>
+        <OsdsText>
+          <span className="text-sky-900 text-xs font-semibold">
+            {t('account-disable-2fa-file-input-attachments')}
+          </span>
+        </OsdsText>
```

Comment:
Why do you need the span inside the ODS text ?
Also, is there any impact on 2FA pages display ?

Reply:
i took the opportunity to add this text in the ods components, it has no impact on the display, as I compared the two

Reply:
But why do we need a span inside the ODS text ? wouldn't the text component be enough ?

Reply:
because i wanted it to look as close as possible to the previous one